scripts/bootstrap.sh

```diff
@@ -119,6 +119,8 @@ if $installing; then echo "$(date -I) -- installed $version" >> /root/events.log
 if $upgrading; then echo "$(date -I) -- upgraded to $version" >> /root/events.log; fi
 
 
+# BUG: during a minion's re-mastering the `master: ...` value may get reset if the instance is 
+# updated by another process before the old master is turned off.
```

Comment:
doing that programmatically would be a valuable addition to builder

Reply:
terraform has a locking system which may help, but I don't know if it can be surfaced to the user with acquire/release primitives https://www.terraform.io/docs/state/locking.html

Reply:
this is actually a severe bug and I misrepresented it. It should read "during the re-mastering transition", meaning, while there is more than one master server active any update will wipe out the master ip info and replace it with the IP of the oldest master-server

Reply:
if we stored the master server IP in the buildvars as well, it might solve this elegantly.

Reply:
It looks like all builds running today will re-master the minion to the old server then. Should block further migrations because they would be easily rolled back?